Wowie! Look at all these changes.

DoctorOctopusTheOctopi opened the pull request:
_Mapping Updates_

- Mine cordoned off for safety reasons.
- Geminus City has had its nuclear weapons removed after concerns brought up by ONI.
- Slums given fronts in which illegal business can be conducted as though it is legal.
- GCPD Armory remodeled, made more secure, organized.
- GCPD Storage Room emptied and refilled with various storage-related items.
- Redesigns the Detective's Office.
- Redesigns the Forensic Scientist's Lab, adds a morgue, full lab, ect.
- Research and Development has been restored to GC's Science Bay.

_Coding Updates_

- Makes R&D more halo-lore friendly.
- Removes the items that got it mapped out in the first place, no more laser guns.
- Project Theta instituted with Lore Master The Pillow's approval.
- Adds a new GCPD Officer Alt-Title, GCPD Forensic Scientist.
- Fixed a few bugs for Desolane.

DoctorOctopusTheOctopi committed:
Merge pull request #5 from HaloSpaceStation/alpha

Local Repo Update 3-28-2018

DoctorOctopusTheOctopi committed:
Merge pull request #6 from HaloSpaceStation/alpha

Local Repo Update 3-29-2018

DoctorOctopusTheOctopi committed:
R&D Changes

- Makes R&D more halo-lore friendly.
- Removes the items that got it mapped out in the first place, no more laser guns.

DoctorOctopusTheOctopi committed:
Geminus City Changes

- Mine cordoned off.
- Colony lost nuke/keycard authentication disk/swipers
- Slums given fronts in which illegal business can be conducted as though it is legal
- Dead ends in slums increased.
- Project Theta instituted with Lore Master The Pillow's approval.
- Research and Development has been restored to GC's Science Bay.
- GCPD Armory remodeled, made more secure, organized.
- GCPD Storage Room emptied and refilled with various storage items.

DoctorOctopusTheOctopi committed:
Finishing Touches

-Redesigns the Detective's Office
-Redesigns the Forensic Scientist's Lab, adds a morgue, full lab, ect.
-Adds a new GCPD Officer Alt-Title
-Fixed a few bugs for Desolane.

DoctorOctopusTheOctopi commented:
I'm gonna go to sleep and fix any Travis errors tomorrow. Its late.. I'm going to add a DNM tag to it just so no one merges it, if Travis approves you can merge it. 🐙

DoctorOctopusTheOctopi committed:
Travis Fixes

Oh great and wise Travis, This will be the last time I attempt to appease you until I wake up tommorow morning with my fresh cup of coffee. Please hath mercy on my poor soul. :octopus:

Desolane commented:
Apparently the dead chef is throwing up issues and something is wrong with the changelog. Otherwise things look fine with the changed files.

DoctorOctopusTheOctopi committed:
Fresh Coffee Travis Fixes

Mhm.. :coffee: This should fix the issues Desolane pointed out and hopefully fix the changelog issues. If Travis doesn't like it I'll octoslap the heck out of him. :angry:
